.ruby-version

```diff
@@ -1 +1 @@
-2.7.1
```

Comment:
why?

Reply:
Because it's the latest patch version. But yes, not really related, I know 😅